airbyte-webapp/src/utils/app.ts

```diff
@@ -0,0 +1 @@
+export const isCloudApp = () => process.env.REACT_APP_CLOUD || window.CLOUD === "true";
```

Comment:
Since this is now extracted into a utility, could we maybe also replace https://github.com/airbytehq/airbyte/blob/master/airbyte-webapp/src/index.tsx#L22 by using this method instead.